R/utils-json.R

```diff
@@ -53,15 +53,32 @@ getMinStage <- function(ts, ...){
 }
 
 #finds if the plot data has any zero values
-zeroValues <- function(dataList, val_nm){    
-  logList <- lapply(dataList, function(x) {any(na.omit(x[[val_nm]]) == 0)})
-  logVector <- any(unlist(unname(logList)))
+zeroValues <- function(data, val_nm){ 
```

Comment:
can this just take in the timeseries instead of data and val_nm? Have the caller pass data[[val_nm]]?